chore(docker/holesky): cleanup docker compose files

thedevbirb opened the pull request:
This PR tries to align some diffs we have between the code on GitHub and our setup on remotebeast, so that we don't have conflicts and headaches when pulling new changes.

Changelog:
- builder -- bump image; on remotebeast it is running a newer one
- helix -- on remotebeast we're running also the website service, which is missing on github
- networks -- unify networks on a single `bolt-holesky-default`
- format run-builder and run-bn

thedevbirb committed:
chore(docker/holesky): add website

thedevbirb committed:
chore(docker/holesky): add other relays when setting up builder

thedevbirb committed:
chore(docker/holesky): fmt bn

thedevbirb committed:
chore(docker/holesky): cleanup compose files

thedevbirb committed:
chore(docker/holesky): bump builder image

mempirate commented on testnets/holesky/docker-compose.yml:
```diff
@@ -82,7 +76,7 @@ volumes:
 networks:
-  bolt-default:
+  bolt-holesky-default:
     driver: bridge
```
You will have to add `name: bolt-holesky-default` here. Otherwise docker compose will prefix it with some bs and the name will differ in `docker-compose.pbs.yml`

thedevbirb replied:
Updated this to drop all references to custom networks, after internal discussion we will use `host.docker.internal` as DNS and adding:
```yaml
    extra_hosts:
      - host.docker.internal:host-gateway
```
On every service that needs to communicate with others which expose a port on the host machine

thedevbirb committed:
chore(docker/holesky): remove custom networks, use host.docker.internal

This is available on Linux on newer version of docker by adding
extra_host:
- host.docker.internal:host-gateway
on services where you need to use that DNS to access the host machine

merklefruit commented on testnets/holesky/docker-compose.yml:
```diff
@@ -71,21 +66,10 @@ services:
-      - bolt-default
-    depends_on:
-      - bolt-prometheus-holesky
+      - bolt-holesky-default
```
what is this network? i thought you said we don't use this anymore

merklefruit reviewed (changes requested):
Looks good, just one small q!

mempirate reviewed (approved):
LGTM, let's merge when tested
